src/locale/locales/pt-BR/messages.po

```diff
@@ -4538,7 +4538,7 @@ msgstr "Sua escolha será salva, mas você pode trocá-la nas configurações de
 
 #: src/screens/Onboarding/StepFollowingFeed.tsx:61
 msgid "Your default feed is \"Following\""
-msgstr ""
+msgstr "Seu feed inicial é o \"Seguindo\"\"
```

Comment:
I think there might be an extra \ at the end here? 🤔

Reply:
Nice catch!